src/Page.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Counter from './Counter';
+import Buttons from './Buttons';
```

Comment:
```suggestion
import React from 'react';

import Counter from './Counter';
import Buttons from './Buttons';
```

범용 라이브러리, 컴포넌트 이런식으로 의미상으로 구분될 때는 빈 줄을 넣어서 구분해주시면 좋아요.
빈 줄 코드 하나에도 의미를 담아 코드를 작성하신다 생각하시면 도움이 되실거에요.

Reply:
넵 다음부터는 구분하겠습니다~!